languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10085,6 +10085,17 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rule id="ADJ_AVANT_OU_ARRIERE" name="adj avant ou arrière" default="temp_off">
+      <pattern>
```

Comment:
Can you confirm that none of the examples found by the rule editor are false positives?
Especially "lignes arrières" seems to be very popular.
Let's put a `<url>` in there, too. How about https://www.larousse.fr/dictionnaires/francais/arri%c3%a8re/5400 ? It says there "L'adjectif est invariable".

Reply:
It isn't a FP as it means "les lignes qui sont à l'arrière (du terrain...)". I put another url taking into account both "avant" and "arrière" and explaining the rule.